x/genutil/client/cli/init.go

```diff
@@ -89,6 +89,9 @@ func InitCmd(mm genesisMM) *cobra.Command {
 			default:
 				chainID = fmt.Sprintf("test-chain-%v", unsafe.Str(6))
 			}
+			if config.RootDir == "" {
+				config.RootDir = clientCtx.HomeDir
+			}
```

Comment:
https://github.com/cosmos/cosmos-sdk/blob/25aea8af8a52823553ddc34bd91f46857d7a54dd/server/util_test.go#L462

The `init` command did not read into the root(hoom) directory during unit test `TestEmptyMinGasPrices`, causing the file to be created in the current directory